conf/app.routes

```diff
@@ -5,7 +5,7 @@
 GET        /assets/*file                                            controllers.Assets.at(path="/public", file)
 GET        /language/:language                                      uk.gov.hmrc.language.ChangeLanguageEndpoint.switchToLanguage(language: String)
 
-GET        /start                                                   uk.gov.hmrc.personaldetailsvalidation.endpoints.PersonalDetailsValidationStartController.start(completionUrl: CompletionUrl, origin: Option[String], failureUrl: Option[CompletionUrl] ?= None)
+GET        /start                                                   uk.gov.hmrc.personaldetailsvalidation.endpoints.PersonalDetailsCollectionController.showPage(completionUrl: CompletionUrl, origin: Option[String], failureUrl: Option[CompletionUrl] ?= None)
```

Comment:
This endpoint now looks identical to GET /personal-details
Which basically means that /start will show the page and that endpoint is no longer needed.
But I'm concerned that that will change the URLs in the journey.  I would prefer it if you just kept PersonalDetailsValidationStartController.start() and just make that redirect to GET /personal-details ?

Reply:
later on we can maybe remove that endpoint, but for now, let's keep the flow the same?